README.md

```diff
@@ -49,6 +49,7 @@ Usage:  <br>
 |`--ont=FILE ...` | validate against the OWL-formatted ontolog(ies) at the specified filename(s) |
 |`--nist` | validate against the NIST restrictions |
 |`--nist-ta3` | validate against the NIST hypothesis restrictions (implies `--nist`) |
+|`--hypothesis-max-size=num` | Specify the maximum size of a hypothesis file in MB when validating against the NIST hypothesis restrictions (`--nist-ta3`). Default is 5 |
```

Comment:
It looks like picocli uses the convention of putting the parameter in angle brackets (e.g., `--hypothesis-max-size=<hypothesisMaxSize>`).  We should probably make the README match the programmatic usage info.

But I'm certainly open to discussion on the matter.

Reply:
Updated.